tests/test_ft_misc.py

```diff
@@ -101,6 +136,25 @@ def test_inline_code():
     assert o.get4() == 4
 
 
+def test_cpp_code_with_constant():
+    o = ft.InlineCode()
+    assert o.cpp_code_with_constant() == 4
+
+
+#
+# operators.h
+#
+
+
+def test_operators_eq():
+    o1 = ft.HasOperator(1)
+    o1a = ft.HasOperator(1)
+    o2 = ft.HasOperator(2)
+
+    assert o1 == o1a
+    assert not (o1 == o2)
```

Comment:
Should this test [`__ne__`](https://docs.python.org/3/reference/datamodel.html#object.__ne__) as well/instead?